src/main/java/com/gravity9/mongocdc/logging/LoggingUtil.java

```diff
@@ -0,0 +1,36 @@
+package com.gravity9.mongocdc.logging;
+
+import com.gravity9.mongocdc.MongoConfig;
+import org.slf4j.MDC;
+
+import java.util.concurrent.atomic.AtomicInteger;
+
+public final class LoggingUtil {
+    private static final String MDC_KEY_NAME = "workerId";
+    private static final AtomicInteger managerCount = new AtomicInteger();
+
+    private LoggingUtil() {
+    }
+
+    public static String createManagerId(MongoConfig config) {
+        return " cdcm_" + config.getDatabaseName() + "_" + config.getCollectionName() + "_" + managerCount.getAndIncrement();
```

Comment:
why leading space?

Reply:
I could have put this space in log4j2.xml, but for classes other than worker and manager there will double space in logs when workerId is empty

Reply:
It would be better to remove it here and add it during formatting for logs, the space looks odd here.

Reply:
Space moved to log4j2.xml. I was trying to solve this using https://logging.apache.org/log4j/2.x/manual/layouts.html#VariablesNotEmpty, to add this space only when the context is present, but this function refuses to work as I would imagine. I guess we will have to live with double space in the 'non-context' logs